src/cascadia/TerminalApp/Tab.cpp

```diff
@@ -8,13 +8,15 @@
 #include "Tab.g.cpp"
 #include "Utils.h"
 #include "ColorHelper.h"
+#include "../../types/inc/utils.hpp"

```

Comment:
Can we not? I think we're trying to put new stuff into TIL. I'm also not a big fan of including the `::Microsoft::Console` namespace in the Terminal App. We're trying to break from that.

Reply:
Yea I just stuck it where I found the original `CoalesceOptionals`. I can move it to `til` in this PR, or make another for specifically that, whichever you prefer @miniksa